playwright-tests/tests/editor.spec.js

```diff
@@ -276,4 +276,84 @@ test.describe("?page=projects&tab=editor", () => {
       await expect(warningText).toBeVisible();
     });
   });
+
+  test.describe("Import project from NEAR Catalog", () => {
+    test.beforeEach(async ({ page }) => {
+      await page.goto(`/${ROOT_SRC}?page=projects&tab=catalogImport`);
+    });
+    test.use({
+      storageState: "playwright-tests/storage-states/wallet-connected.json",
+    });
+
+    // should have visible projects
+    test("should have visible projects", async ({ page }) => {
+      const projectHeading = page.getByRole("heading", {
```

Comment:
What's going to happen a week from now when new projects have been added to the catalog and "Race of Sloths" is no longer visible on first page?

Need to search first, then get by role

Reply:
Sure.